redhat-developer/developers.redhat.com
# DEVELOPER 5402 - add social links to articles

### JIRA Issue Link
* https://issues.jboss.org/browse/DEVELOPER-5402

### Verification Process
1. Visit an article
2. Verify social links match mockups
3. Verify social links share to each individual site

### Screenshots
![image](https://user-images.githubusercontent.com/10933938/46226696-1e0d5380-c32b-11e8-8cee-60f1cc7ccb5b.png)
![image](https://user-images.githubusercontent.com/10933938/46227243-9d4f5700-c32c-11e8-8ad4-0bec437885b7.png)
